__tests__/Company_test.re

```diff
@@ -7,17 +7,17 @@ describe("BsFaker.Company", () => {
     expect(Js.Array.isArray(Company.suffixes())) === true
   );
   describe("companyName", () => {
-    test("accepts optional args", () =>
-      expect(Js.typeof(Company.companyName())) === "string"
-    );
+    // test("accepts optional args", () =>
```

Comment:
You are correct. I didn't realized that these calls were redundant. Since the incoming value was an option, i assumed it had to return an option. I made the recommended edits and it still compiles AND the tests pass.